crates/uv-build-frontend/src/error.rs

```diff
@@ -71,15 +72,31 @@ pub enum Error {
     Virtualenv(#[from] uv_virtualenv::Error),
     #[error("Failed to run `{0}`")]
     CommandFailed(PathBuf, #[source] io::Error),
-    #[error("{message} ({exit_code})\n--- stdout:\n{stdout}\n--- stderr:\n{stderr}\n---")]
+    #[error("{message} ({exit_code})\n\n{}\n{stdout}\n\n{}\n{stderr}\n", "[stdout]".red(), "[stderr]".red())]
     BuildBackendOutput {
         message: String,
         exit_code: ExitStatus,
         stdout: String,
         stderr: String,
     },
-    /// Nudge the user towards installing the missing dev library
-    #[error("{message} ({exit_code})\n--- stdout:\n{stdout}\n--- stderr:\n{stderr}\n---")]
+    #[error("{message} ({exit_code})\n\n{}\n{stdout}\n", "[stdout]".red())]
+    BuildBackendStdout {
```

Comment:
Can we use a wrapper type with `stdout: Option<String>, stderr: Option<String>` that `impl Display` instead of duplicating this?